src/components/admin/ArticleListController.jsx

```diff
@@ -117,7 +117,11 @@ export default class ArticleListController extends FalcorController {
   render() {
     if (this.state.ready) {
       // If trying to access inacessible page, redirect to page 1
-      if (!this.state.data.articles.byPage) {
+      if (
```

Comment:
Use `_.has(object, path)` here (https://lodash.com/docs/4.17.5#has)

Reply:
Such a nice short documentation link btw, I love it :P